src/site/stages/build/plugins/modify-dom/check-broken-links/index.js

```diff
@@ -15,6 +15,16 @@ module.exports = {
     const fileNames = Object.keys(files);
     this.allPaths = new Set(fileNames);
     this.brokenPages = [];
+
+    this.logFile = path.join(
+      __dirname,
+      '../../../../../../../logs',
```

Comment:
Is there any way we can use an absolute path here?

Reply:
This does produce an absolute path

![image](https://user-images.githubusercontent.com/1915775/116579569-82786100-a8e0-11eb-97fe-5db459f66ead.png)

Is that what you meant?

Reply:
The `../../../../../../../logs` part is what concerns me. But if there's no better way, so be it!

Reply:
It's the most precise way I could think because there isn't a way AFAIK to consistently start from the project root because yeah I know that `../` isn't pretty